rust/src/accuracy/mod.rs

```diff
@@ -237,8 +409,8 @@ pub mod test {
         let accuracy = 1.0;
         let theoretical_alpha = 0.05;
         let scale = accuracy_to_laplacian_scale(accuracy, theoretical_alpha)?;
-        let base_laplace = make_base_laplace::<AllDomain<f64>>(scale, None)?;
-        let n = 50_000;
+        let base_laplace = make_base_laplace::<AllDomain<f64>>(scale, Some(-100))?;
+        let n = 500_000;
```

Comment:
These are a little slow at 500K, do you want this so big?

Similar for `test_empirical_discrete_gaussian_accuracy`.